debian/Makefile

```diff
@@ -51,13 +52,15 @@ ifeq ($(APPLY_PATCHES),yes)
 	git reset --hard HEAD
 	cat debian/patches/series | xargs -iPATCH bash -c 'patch -p1 < debian/patches/PATCH'
 endif
+	touch apply-patches
```

Comment:
out of curiosity why are we touching these files?

Reply:
TL:DR: I put these here so when we execute:

```
# Build Deb
git checkout -b debian-0.14.0-ksqldb origin/0.14.0-ksqldb
dch --newversion "6.1.0-0.ksqldb0.14.0" "Release version 6.1.0-0.ksqldb0.14.0"
dch --release --distribution unstable ""
git commit ...
gbp buildpackage -us -uc --git-debian-branch="debian-0.14.0-ksqldb" --git-upstream-tree=0.14.0-ksqldb --git-builder="debuild --set-envvar=VERSION=0.14.0  -i -I"
# Build RPM
make RPM_VERSION=6.1.0 VERSION=0.14.0 PACKAGE_TYPE=rpm REVISION=ksqldb -f debian/Makefile rpm
# Build ZIP & TGZ
make VERSION=0.14.0 PACKAGE_TYPE=archive -f debian/Makefile archive
```

In that order, it doesn't have to execute the `build` target every time I call `make` (`gbp buildpackage` uses Make under the hood to build the Debian package), as in we build once (mvn install) then re-use that built-out directly subsequent times. 

Longer:

GNU Make under the hood looks for a file named the target, if it doesn't find said file, then it executes the rules (under the target) to generate that target, then moves to the next target that required this target to be ran. If the file has a newer timestamp than its upstream targets, it re-executes the target block. Make works this way because it was designed for doing incremental builds for C/C++ code - as in you'd see `%.o: %.c` targets whose block would run GCC to compile a source file (anything ending in .c) into a object file (a gcc .o object file), which would then be used up higher as a build goal to build a binary executable or a shared library. This way you change a single `.c` file, and Make (based on timestamps) would deduce what "things" to rebuild and re-link. 

Then you have examples like this, where it completely discards that feature of Make, and this is a minor "hack" to get things working. The alternative is that it gets rebuilt every time, leading to a (future) longer build & release Pipeline for KSQLDB. There is a minor cost: Developers will need to run `make clean` in order to get a fresh new build (which I'll add to the clean target), but I doubt that many developers use these code paths often, and are mostly here for Confluent's purposes.